src/ResourceManager/KeyVault/Commands.KeyVault/Commands/SetAzureKeyVaultCertificateIssuer.cs

```diff
@@ -107,10 +107,11 @@ public class SetAzureKeyVaultCertificateIssuer : KeyVaultCmdletBase
         /// </summary>
         [Parameter(Mandatory = true,
                    ParameterSetName = ByValueParameterSet,
+                   ValueFromPipeline = true,
                    ValueFromPipelineByPropertyName = true,
                    HelpMessage = "Specifies the certificate issuer to set.")]
         [ValidateNotNullOrEmpty]
-        public KeyVaultCertificateIssuer Issuer { get; set; }
+        public PSKeyVaultCertificateIssuerIdentityItem Issuer { get; set; }
```

Comment:
@maddieclayton do we want to name this `InputObject` with an alias to `Issuer`?

Reply:
Done